helm/circles-subgraphs/subgraph-job-STG.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    kompose.cmd: kompose convert -o circles-infra-suite --volumes persistentVolumeClaim -c 
+    kompose.version: 1.21.0 (992df58d8)
+    # This is what defines this resource as a hook. Without this line, the
+    # job is considered part of the release.
+    helm.sh/hook: post-install
+    helm.sh/hook-delete-policy: hook-succeeded
+  creationTimestamp: null
+  labels:
+    io.kompose.service: subgraph
+  name: subgraph-130-v140
+spec:
+  template:
+    metadata:
+      annotations:
+        kompose.cmd: kompose convert -o circles-infra-suite --volumes persistentVolumeClaim -c
+        kompose.version: 1.21.0 (992df58d8)
+      creationTimestamp: null
+      labels:
+        io.kompose.service: subgraph
+    spec:
+      imagePullSecrets:
+        - name: circles-registry
+      containers:
+      - env:
+        - name: GRAPH_ADMIN_NODE_ENDPOINT
+          valueFrom:
+              configMapKeyRef:
+                key: GRAPH_ADMIN_NODE_ENDPOINT
+                name: env
+        - name: SUBGRAPH_NAME
+          value: circlesubi/circles-ubi-stg
+        - name: SUBGRAPH_NETWORK
+          value: chiado
+        - name: PROXY_FACTORY_ADDRESS
+          value: 0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2
+        - name: PROXY_FACTORY_ADDRESS_CRC
+          value: 0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2
+        - name: START_BLOCK
+          value: "0"
+        - name: HUB_ADDRESS
+          value: 0xdbF22D4e8962Db3b2F1d9Ff55be728A887e47710
+        - name: IPFS_NODE_ENDPOINT
+          valueFrom:
+              configMapKeyRef:
+                key: IPFS_NODE_ENDPOINT
+                name: env
+        - name: postgres_host
+          value: postgres
+        - name: postgres_user
+          value: graph-node
+        - name: postgres_pass
+          value: let-me-in
+        - name: postgres_db
+          value: graph-node
+        image: registry.digitalocean.com/circles-registry/circles-subgraph:v1.4.0
```

Comment:
there is the new version `v2.1.0`, are we using it in prod?

Reply:
not yet

Reply:
I can upgrade here yes, and then we will do prod next week ^^

Reply:
As long as it's consistent with prod, it's fine :)

Reply:
thanks for pointing this out, I deployed the release version and realised that there was a mistake and caught it before having deployed in PRD!